bin/crew

```diff
@@ -1703,6 +1711,9 @@ def archive_package(crew_archive_dest)
     end
   end
   system "sha256sum #{crew_archive_dest}/#{pkg_name} > #{crew_archive_dest}/#{pkg_name}.sha256"
+  # Copy package file for the successfully generated package to CREW_LOCAL_REPO_ROOT .
+  FileUtils.cp "#{CREW_PACKAGES_PATH}/#{@pkgName}.rb", "#{CREW_LOCAL_REPO_ROOT}/packages/" unless CREW_PACKAGES_PATH.include?(CREW_LOCAL_REPO_ROOT)
+  puts "The package file for #{@pkgName} used has been copied to #{CREW_LOCAL_REPO_ROOT}/packages/".lightblue
```

Comment:
How about this?
```
if CREW_LOCAL_REPO_ROOT.include(CREW_LIB_PATH)
  FileUtils.cp "#{CREW_PACKAGES_PATH}/#{@pkgName}.rb", "#{CREW_LOCAL_REPO_ROOT}/packages/"
else
  FileUtils.cp "#{CREW_LOCAL_REPO_ROOT}/packages/#{@pkgName}.rb", CREW_PACKAGES_PATH.to_s
end
```

Reply:
This extra condition is no longer necessary, since you can no longer build when in `/usr/local/lib/crew/packages`. (You can still do a `crew install -s package` from there...

Reply:
This line is still incorrect for my workflow.  I always make changes in my local repo so any package file updates will be copied back to the crew repo.

Reply:
I dropped it behind the `-f` flag so you don't have to deal with it.